src/components/header/sign-markup.test.tsx

```diff
@@ -0,0 +1,43 @@
+import { render, screen } from '@testing-library/react';
+import { withHistory, withStore } from '../../mock-component';
+import { AuthorizationStatus } from '../../constants';
+import SignMarkup from './sign-markup';
+
+describe('Component: SignMarkup', () => {
+  const signMarkupContainerTestId = 'auth-markup-container';
+  const unsignMarkupContainerTestId = 'no-auth-markup-container';
+
+  it('should render correct when user not authorized', () => {
```

Comment:
Ну и везде по коду аналогично

```suggestion
  it('should render correctly when the user is not authorized', () => {
```